cmake/Common.cmake

```diff
@@ -31,10 +31,14 @@ endforeach()
 
 
 # -----------------------------------------------------------------------------
-# CPP11 
+# CPP11
 # -----------------------------------------------------------------------------
-set (CMAKE_CXX_STANDARD 11)
-set (CMAKE_CXX_STANDARD_REQUIRED TRUE)
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
is CMAKE_CXX_STANDARD always >= 11 ?

Reply:
Well, the user could explicitly provide `-DCMAKE_CXX_STANDARD=98`, bu t I have never seen that.
I can throw a fatal error if defined and less than 11, will update.